pkg/sentry/fs/host/tty.go

```diff
@@ -43,6 +43,8 @@ type TTYFileOperations struct {
 	// fgProcessGroup is the foreground process group that is currently
 	// connected to this TTY.
 	fgProcessGroup *kernel.ProcessGroup
+
+	termios linux.KernelTermios
```

Comment:
Since we've added a termios, I think we should give it the same default values Linux does. Can you please add an initial value for `termios` below in newTTYFile():
```
termios: linux.DefaultSlaveTermios,
```

Reply:
Great idea. I was considering to obtain it from host which seems too heavy. Your solution looks great to me. Thanks!